app/Http/Controllers/ProjectController.php

```diff
@@ -116,47 +124,86 @@ public function update(Request $request, $id)
     public function destroy($id)
     {
       $project = Project::find($id);
+      $project->members()->detach();
       $project->delete();
 
-      return response()->json('Project Deleted Successfully.');
+      return response()->json(['status' => true, 'message' => 'Project Deleted Successfully.']);
     }
 
     public function assignMember(Request $request)
     {
-        $idMember = $request->get('member_id');
-        $idProject = $request->get('project_id');
-        $project = Project::find($idProject);
-        $member = Member::find($idMember);
-        if ($member && $project) {
-            DB::table('project_member')->insert(
-            [
-                'project_id' => $idProject,
-                'member_id' => $idMember,
-                'role' => $request->get('role'),
-                'created_at' => date('Y-m-d H:i:s'),
-                'updated_at' => date('Y-m-d H:i:s')
-            ]);
-            return response()->json('Assign Successfully.');
+      $rules = [
+      'project_id' => 'required',
+      'member_id' => 'required',
+      'role' => 'required'
+      ];
+      $result = $request->validate($rules);
+      $idMember = $request->get('member_id');
+      $idProject = $request->get('project_id');
+      $project = Project::find($idProject);
+      $member = Member::find($idMember);
+      if ($member && $project) {
+        DB::table('project_member')->insert(
```

Comment:
Please write database logic in a `Model` and not in the `Controller`

Reply:
thanks  i willl change it